src/main/java/com/jfrog/ide/idea/scan/data/Rule.java

```diff
@@ -15,6 +16,9 @@ public class Rule {
     @JsonProperty("fullDescription")
     private Message fullDescription;
 
+    @JsonProperty("properties")
+    private RuleProperties properties;
```

Comment:
not all rules have properties. should the field itself be optional

Reply:
is that a question? yes - it should be optional...